tree/ntuple/v7/test/ntuple_types.cxx

```diff
@@ -37,7 +37,10 @@ TEST(RNTuple, TypeNameNormalization)
    EXPECT_EQ("std::int32_t", RFieldBase::Create("f", "signed").Unwrap()->GetTypeName());
    EXPECT_EQ("std::map<std::int32_t,std::int32_t>", RFieldBase::Create("f", "map<int, int>").Unwrap()->GetTypeName());
 
-   EXPECT_TRUE(RFieldBase::Create("f", "std::int32_t").Unwrap()->GetTypeAlias().empty());
```

Comment:
You probably want to keep this newly added test that `std::int32_t` has an empty type alias